### DISCUSSION TITLE
Is there an easy way to edit checkbox icon?

### DISCUSSION THREAD

**Discussion by zirho** [saadeghi/daisyui#2300] [Q&A] [answered]
I see `src/components/styled/checkbox.css`

https://github.com/saadeghi/daisyui/blob/366e181b9a699c0eb3960aecba82d7bf37201feb/src/components/styled/checkbox.css#L13C1-L24

It has icon defined in linear-gradient.
is there an easy way to customize the code just a little bit?

I am trying to round the edges of the check icon.

Thanks in adance.

**Comment by saadeghi** [comment, marked as answer]
An easy way to use custom icons is using these class names:  
```
checked:bg-[url('https://www.svgrepo.com/show/410398/check.svg')] bg-cover
```

Example: https://play.tailwindcss.com/3pQjUsOcJk

**Reply by zirho** [reply]
thank you!
